### ISSUE TITLE
Security Group Update

### ISSUE THREAD

**Issue by ayberkcal** [ContainX/openstack4j#973]
Hi,
I would like to update security group name and description but I can not find implementation in openstack4j. Just implemented get/delete/create/list. Am I missing something or update do not implemented?
Thank you.

**Comment by auhlig** [comment]
Do you mean [this](https://github.com/ContainX/openstack4j/blob/6c3451d7073282535d5982b6bcc72eba28b7d0ba/core/src/main/java/org/openstack4j/api/networking/SecurityGroupService.java)?

**Comment by ayberkcal** [comment]
Yes, that is what I am saying :) Thank you I will implement as soon as possible.

**Comment by auhlig** [comment]
Cool. Looking forward :)
